cfme/tests/infrastructure/test_advanced_search_providers.py

```diff
@@ -146,7 +141,7 @@ def cleanup():
 @pytest.mark.requires("test_can_open_advanced_search")
 @pytest.mark.meta(blockers=[1168336])
 def test_filter_save_and_load_cancel(request, providers, ssh_client):
-    pytest.sel.force_navigate("infrastructure_providers")
+    prov = providers
```

Comment:
Can we please change the name of this fixture to ```single_provider```, as providers seems weird and the cast here is redundant.